docs/get-started/labs/labs-prerequisites.md

```diff
@@ -6,36 +6,40 @@ sidebar_label: Lab Prerequisites
 
 ## Prerequisites
 
-1. DNS is configured. See [here](get-started/installation/overview.md).
+1. DNS is configured. See [here](get-started/installation/overview.md). Not needed when using the LKE automatic installation.
 
-2. Platform Applications that are required for the following labs are enabled:
+2. Prometheus is enabled on the platform level.
+
+2. Enable additional App on the platform level to support the following labs:
 
 | Lab                                         | App                                 |
 |---------------------------------------------|-------------------------------------|
-| Build images                                | Harbor                              |
+| Create container images                     | Harbor                              |
 | Trigger builds                              | Harbor                              |
-| Push images to harbor                       | Harbor                              |
+| Manually Push images to Harbor              | Harbor                              |
 | Scan running containers for vulnerabilities | Prometheus, Grafana, Trivy Operator |
 | View container logs                         | Loki, Grafana                       |
-| View container metrics                      | Prometheus                          |
-| Monitor services                            | Alertmanager                        |
-| Using custom metrics                        | Prometheus                          |
 | Tracing with Open Telemetry                 | Loki, Otel, Tempo                   |
 | Create a RabbitMQ cluster                   | RabbitMQ                            |
 
-
+:::info
 For the [Tracing with Open Telemetry](use-otel.md) Lab, tracing needs to be configured in the `Istio` and `Nginx Ingress` apps.
+:::
+
+3. A team called `labs` is created with `Dashboards` and `Alerts` enabled. See [here](docs/for-ops/console/teams.md) how to create Teams.
 
-3. A team called `labs` is created with `Alertmanager` activated. See how you can create teams on: [Platform - Teams](docs/for-ops/console/teams.md).
+4. A user account is created and added to the `labs` Team. In the labs we'll be using the user `labs-user@labs.com`. See [here](docs/for-ops/console/usermgnt.md)how to create users and assign them to Teams.
 
-4. A user account is created and added to the `labs` Team. In the labs we'll be using the user `labs-user@example.com`. See how you can create users on: [Platform - User Management](docs/for-ops/console/usermgnt.md).
+:::note
```

Comment:
We are missing a prerequisite to set API URL in cluster settings page.

Reply:
![image](https://github.com/user-attachments/assets/2cb1358d-6800-40a6-bcc0-75cc7ede8772)

Reply:
Added